consensus/polybft/stake_manager.go

```diff
@@ -116,38 +113,25 @@ func (s *stakeManager) PostBlock(req *PostBlockRequest) error {
 
 	stakeMap := fullValidatorSet.Validators
 
-	updatedValidatorsStake := make(map[types.Address]struct{}, 0)
+	s.logger.Debug("full validator set before", "block", fullValidatorSet.BlockNumber, "data", stakeMap)
```

Comment:
Should we specify that `stakeMap` is not ordered (either change the log message or add a comment in the code). It can come as a surprise that order is not the same, but in fact we are iterating through `map`.
```suggestion
	s.logger.Debug("full validator set before", "block", fullValidatorSet.BlockNumber, "validators (unordered)", stakeMap)
```

Reply:
fixed https://github.com/0xPolygon/polygon-edge/pull/1461/commits/758c3961fb1ae40fb85d7e8a871386ef4479658c

Reply:
printed validators are now ordered (negative and zero stake validators are removed)